Model/Sales/Repository/PackageRepository.php

```diff
@@ -238,6 +238,26 @@ public function isDeliveryOptionsDisabled($products)
         return $this;
     }
 
+    /**
+     * @param array $products
+     *
+     * @return int|null
+     */
+    public function getProductDropOffDelay(array $products): ?int
+    {
+        $heightsDropOffDelay = null;
+
+        foreach ($products as $product) {
+            $dropOffDelay = $this->getAttributesProductsOptions($product, 'dropoff_delay');
+
+            if ($dropOffDelay >= $heightsDropOffDelay) {
```

Comment:
het heeft weinig zin om hier `>=` te doen, niet dat het functioneel veel uitmaakt maar `>` is logischer

Reply:
true, had in eerste instantie de method op een andere mannier gedaan (done)